Documentation/data-cleanup/python-script/program_deletion.py

```
@@ -0,0 +1,157 @@
+
+    payload = {
+        "identifier": USERNAME,
+        "password": PASSWORD
+    }
+    headers = {
+        "origin": program_deletion_config.get('origin', ''),
+        "Content-Type": "application/json"
+    }
+
+    try:
+        res = requests.post(LOGIN_URL, json=payload, headers=headers)
```

Ruff (0.15.5)

[error] Probable use of `requests` call without timeout

(S113)